students/DennisLee/lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,19 @@
+def print_fizz_buzz() -> None:
+    """
+    Print numbers 1 to 100, but 'Fizz' or 'Buzz' for multiples of 3 or 5
+
+    :return:  None.
+    """
+    for j in range(100):
```

Comment:
This can be written `for i in range(1, 101):` and then you don't need the line 8 assignment